src/main/app.ts

```diff
@@ -54,6 +54,7 @@ app.use(cookieParser(config.get('session.encryptionKey')))
 app.use(cookieEncrypter(config.get('session.encryptionKey'), CookieProperties.getCookieConfig()))
 
 app.use('/legal', express.static(path.join(__dirname, 'public')))
```

Comment:
We should remove that at some point since we use separate domain.

Reply:
yup agree, or at least add a redirect